app/javascript/shared/components/Spinner.jsx

```diff
@@ -0,0 +1,18 @@
+import { h } from 'preact';
+
+export const Spinner = () => (
+  <svg
+    className="crayons-spinner"
+    width="18px"
+    height="18px"
+    viewBox="0 0 18 18"
```

Comment:
Maybe we could redo the SVG a little to be within 24px artwork like rest of our icons? Im happy to do so, let me know if you need help with that.

Reply:
I can put it to 24px. I was basing it off the Figma page that you showed me where it had it as 18px for the image uploaded.